how-to/features/instance-recovery.rst

```diff
@@ -13,6 +13,10 @@ To enable Instance Recovery, run the following command:
 
     sunbeam enable instance-recovery
 
+.. note::
```

Comment:
We need to have the label also outlined here, people trying to deploy the feature on MAAS should also see it from here.